core-api/src/test/java/com/optimizely/ab/OptimizelyTest.java

```diff
@@ -3061,6 +3065,11 @@ public void getFeatureVariableValueReturnsDefaultValueWhenNoVariationUsageIsPres
         );
 
         assertEquals(expectedValue, value);
+
+        logbackVerifier.expectMessage(
+            Level.INFO,
+            "Variable \"integer_variable\" is not used in variation \"589640735\", returning default value \"7\"."
```

Comment:
This log message denotes that the code needs to be updated. Please see recommendations to what this message should be. I think it is `Variable <variable name> 's value is not defined. Returning default value <value>`